README.md

```diff
@@ -23,125 +27,246 @@ All LLM responses are cached in the collection specified. If the same exact ques
-                                    "similarity": "dot_product",
-                                    "type": "vector",
-                                    "vector_index_optimized_for": "recall"
-                                }
-                            ]
-                        },
-                        "text": {
-                            "enabled": true,
-                            "dynamic": false,
-                            "fields": [
-                                {
-                                    "index": true,
-                                    "name": "text",
-                                    "store": true,
-                                    "type": "text"
-                                }
-                            ]
-                        }
-                    }
-                }
-            }
-        },
-        "store": {
-            "indexType": "scorch",
-            "segmentVersion": 16
-        }
-    },
-    "sourceType": "gocbcore",
-    "sourceName": "pdf-docs",
-    "sourceParams": {},
-    "planParams": {
-        "maxPartitionsPerPIndex": 64,
-        "indexPartitions": 16,
-        "numReplicas": 0
-    }
+### Set the environment secrets
+
+Copy the `secrets.example.toml` file in `.streamlit` folder and rename it to `secrets.toml` and replace the placeholders with the actual values for your environment.
+
+**For Couchbase Vector Search - Hyperscale/Composite (`chat_with_pdf_query.py`):**
+```toml
+OPENAI_API_KEY = "<open_ai_api_key>"
+DB_CONN_STR = "<connection_string_for_couchbase_cluster>"
+DB_USERNAME = "<username_for_couchbase_cluster>"
+DB_PASSWORD = "<password_for_couchbase_cluster>"
+DB_BUCKET = "<name_of_bucket_to_store_documents>"
+DB_SCOPE = "<name_of_scope_to_store_documents>"
+DB_COLLECTION = "<name_of_collection_to_store_documents>"
+CACHE_COLLECTION = "<name_of_collection_to_cache_llm_responses>"
+AUTH_ENABLED = "False"
+LOGIN_PASSWORD = "<password_to_access_the_streamlit_app>"
+```
+
+**For Couchbase Search (`chat_with_pdf.py`):**
+```toml
+OPENAI_API_KEY = "<open_ai_api_key>"
+DB_CONN_STR = "<connection_string_for_couchbase_cluster>"
+DB_USERNAME = "<username_for_couchbase_cluster>"
+DB_PASSWORD = "<password_for_couchbase_cluster>"
+DB_BUCKET = "<name_of_bucket_to_store_documents>"
+DB_SCOPE = "<name_of_scope_to_store_documents>"
+DB_COLLECTION = "<name_of_collection_to_store_documents>"
+CACHE_COLLECTION = "<name_of_collection_to_cache_llm_responses>"
+INDEX_NAME = "<name_of_search_index_with_vector_support>"
+AUTH_ENABLED = "False"
+LOGIN_PASSWORD = "<password_to_access_the_streamlit_app>"
+```
+
+> **Note:** Couchbase Vector Search approach does not require `INDEX_NAME` parameter.
+
+---
+
+## Approach 1: Couchbase Vector Search (Hyperscale/Composite)
+
+For the full tutorial on Couchbase Vector Search approach, please visit [Developer Portal - Couchbase Vector Search](https://developer.couchbase.com/tutorial-python-langchain-pdf-chat-gsi).
```

Comment:
Note: This URL should be updated once we have a common understanding of the path structure we want to follow.